parsons/ngpvan/people.py

```diff
@@ -360,6 +360,12 @@ def get_person(self, id, id_type='vanid', expand_fields=[
 
         expand_fields = ','.join([json_format.arg_format(f) for f in expand_fields])
 
+        # Removing the fields that are not returned in MyVoters
+        NOT_IN_MYVOTERS = ['codes', 'contribution_history', 'organization_roles']
+
+        if self.connection.db == 0:
```

Comment:
I think you still want this to be `self.connection.db_code == 0` OR `self.connection.db == 'MyVoters'` (I prefer the former).